tests/end_to_end/conftest.py

```diff
@@ -117,6 +117,27 @@ def generate_events(request, metadata):
         ansible_runner.run(**parameters)
 
 
+@pytest.fixture(scope='module')
+def get_manager_ip(request):
+    """Get manager IP.
+
+       Returns:
+            str: Manager IP.
+    """
+    inventory_playbook = [request.config.getoption('--inventory_path')]
+
+    if not inventory_playbook:
```

Comment:
Change this check, because if the inventory_playbook has one item and it is None, this condition will be evaluated to True

Reply:
Done in 9c69f64